README.rst

```diff
@@ -0,0 +1,42 @@
+====================
```

Comment:
Was the conversion necessary?

Reply:
If nothing else, we use rst in all other repositories.